openbas-api/src/main/java/io/openbas/utils/pagination/PaginationUtils.java

```diff
@@ -46,4 +46,26 @@ public static <T, U> Page<U> buildPaginationCriteriaBuilder(
     return findAll.apply(filterSpecifications.and(searchSpecifications), pageable);
   }
 
+  /**
+   * Build PaginationJPA with a specified specifications
+   * @param findAll the find all method
+   * @param input the search inputs
+   * @param specification the specified specification
+   * @param clazz the class that we're looking for
+   * @return a Page of results
+   */
```

Comment:
I think of other developers and I say to myself that we should be more specific on the nomenclature, the doc or the name of the variables to clearly highlight the fact that here the specification given as a parameter replaces the default search specification.

Reply:
I made changes to try to be more explicit